common.lic

```diff
@@ -268,9 +268,10 @@ module DRC
     end
   end
 
-  def hide?
+  def hide?(*hide_type)
+    hide_type.any? ? hide_type = hide_type[0] : hide_type = 'hide' 
```

Comment:
`hide_type: hide` for both of them. The goal is just to safe guard them if they decide to overwrite their `hide_type: stalk` with `hide_type:` in a later file.